solidity/contracts/isms/hook/AbstractHookISM.sol

```diff
@@ -30,15 +30,13 @@ import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Ini
  * @dev In the future, the hook might be moved inside the Mailbox which doesn't require storage mappings for senders.
  *      for more details see https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/2381
  */
-abstract contract AbstractNativeISM is
-    IInterchainSecurityModule,
-    Initializable
-{
+abstract contract AbstractHookISM is IInterchainSecurityModule, Initializable {
     // ============ Public Storage ============
 
     // mapping to check if the specific messageID is from a specific sender
     // @dev anyone can send an untrusted messageId, so need to check for that while verifying
-    mapping(bytes32 => bytes32) public verifiedMessageIds;
+    // @dev anyone can overwrite the mapping for a messageId, and hence need the sender => bool mapping
```

Comment:
nit: Missing a comment that describes the variable (e.g. "Maps sender to messageId to whether or not that sender attested to that message ID on the origin chain")